### ISSUE TITLE
global 'ends' is not callable

### ISSUE THREAD

**Issue by dandare100** [ntop/ntopng#6841]
**Environment**:
* OS name: Rocky Linux 8
* OS version: 8
* Architecture:  x86_64 x86_64 x86_64 GNU/Linux
* ntopng version/revision: dev branch

**What happened**:
Warning on startup :
`
04/Sep/2022 21:40:05 [LuaEngine.cpp:595] WARNING: Script failure [/home/jeff/graft/repos/ntopng/scripts/callbacks/5second/system/timeseries.lua][...ff/graft/repos/ntopng/scripts/lua/modules/ntop_utils.lua:342: global 'ends' is not callable (a nil value)]
`

**How did you reproduce it?**
Just start ntopng.
It also causes tests to fail

**Debug Information**:

ntop logs
```
[jeff@rocky01 ntopng]$ sudo ./ntopng -s -i enp2s0
04/Sep/2022 21:53:28 [Redis.cpp:151] Successfully connected to redis 127.0.0.1:6379@0
04/Sep/2022 21:53:28 [Redis.cpp:151] Successfully connected to redis 127.0.0.1:6379@0
04/Sep/2022 21:53:30 [PcapInterface.cpp:96] Reading packets from enp2s0 [id: 1]
04/Sep/2022 21:53:30 [Ntop.cpp:2584] Registered interface enp2s0 [id: 1]
04/Sep/2022 21:53:30 [main.cpp:344] PID stored in file /var/run/ntopng.pid
04/Sep/2022 21:53:30 [Geolocation.cpp:149] Loaded database GeoLite2-City.mmdb [/usr/share/GeoIP/GeoLite2-City.mmdb][ip_version: 6]
04/Sep/2022 21:53:30 [Geolocation.cpp:116] Running without geolocation support.
04/Sep/2022 21:53:30 [Geolocation.cpp:117] To enable geolocation follow the instructions at
04/Sep/2022 21:53:30 [Geolocation.cpp:118] https://github.com/ntop/ntopng/blob/dev/doc/README.geolocation.md
04/Sep/2022 21:53:30 [HTTPserver.cpp:1412] HTTPS Disabled: missing TLS certificate /home/jeff/graft/repos/ntopng/httpdocs/ssl/ntopng-cert.pem
04/Sep/2022 21:53:30 [HTTPserver.cpp:1414] Please read https://github.com/ntop/ntopng/blob/dev/doc/README.SSL if you want to enable TLS.
04/Sep/2022 21:53:30 [HTTPserver.cpp:1683] Web server dirs [/home/jeff/graft/repos/ntopng/httpdocs][/home/jeff/graft/repos/ntopng/scripts]
04/Sep/2022 21:53:30 [HTTPserver.cpp:1686] HTTP server listening on 3000
04/Sep/2022 21:53:30 [NetworkInterface.cpp:3092] Started host user script hooks loop on interface enp2s0 [id: 1]...
04/Sep/2022 21:53:30 [NetworkInterface.cpp:3036] Started flow user script hooks loop on interface enp2s0 [id: 1]...
04/Sep/2022 21:53:30 [main.cpp:414] Working directory: /var/lib/ntopng
04/Sep/2022 21:53:30 [main.cpp:416] Scripts/HTML pages directory: /home/jeff/graft/repos/ntopng
04/Sep/2022 21:53:30 [Ntop.cpp:467] Welcome to ntopng x86_64 v.5.5.220904 - (C) 1998-22 ntop.org
04/Sep/2022 21:53:30 [Ntop.cpp:477] Built on Rocky Linux release 8.6 (Green Obsidian)
04/Sep/2022 21:53:30 [FlowRiskAlerts.cpp:199] [!] nDPI risk 26/Risky ASN has not been defined in ntopng
04/Sep/2022 21:53:30 [FlowRiskAlerts.cpp:199] [!] nDPI risk 27/Risky Domain Name has not been defined in ntopng
04/Sep/2022 21:53:30 [Ntop.cpp:874] Adding 192.168.8.10/32 as IPv4 interface address for enp2s0
04/Sep/2022 21:53:30 [Ntop.cpp:883] Adding 192.168.8.0/24 as IPv4 local network for enp2s0
04/Sep/2022 21:53:30 [PeriodicActivities.cpp:109] Started periodic activities loop...
04/Sep/2022 21:53:31 [startup.lua:35] Processing startup.lua: please hold on...
04/Sep/2022 21:53:31 [startup.lua:120] [lists_utils.lua:813] Refreshing category lists...
04/Sep/2022 21:53:32 [startup.lua:120] [lists_utils.lua:737] Category Lists (3824 hosts, 18842 IPs, 0 JA3) loaded in 1 sec
04/Sep/2022 21:53:32 [startup.lua:124] Initializing device polices...
04/Sep/2022 21:53:32 [startup.lua:140] Initializing alerts...
04/Sep/2022 21:53:32 [startup.lua:149] Initializing timeseries...
04/Sep/2022 21:53:32 [startup.lua:212] Fetching latest ntop blog posts...
04/Sep/2022 21:53:32 [startup.lua:216] Completed startup.lua
04/Sep/2022 21:53:32 [PeriodicActivities.cpp:167] Found 10 activities
04/Sep/2022 21:53:32 [NetworkInterface.cpp:3258] Started packet polling on interface enp2s0 [id: 1]...
04/Sep/2022 21:53:35 [LuaEngine.cpp:595] WARNING: Script failure [/home/jeff/graft/repos/ntopng/scripts/callbacks/5second/system/timeseries.lua][...ff/graft/repos/ntopng/scripts/lua/modules/ntop_utils.lua:342: global 'ends' is not callable (a nil value)]
04/Sep/2022 21:53:40 [LuaEngine.cpp:595] WARNING: Script failure [/home/jeff/graft/repos/ntopng/scripts/callbacks/5second/system/timeseries.lua][...ff/graft/repos/ntopng/scripts/lua/modules/ntop_utils.lua:342: global 'ends' is not callable (a nil value)]
04/Sep/2022 21:53:45 [LuaEngine.cpp:595] WARNING: Script failure [/home/jeff/graft/repos/ntopng/scripts/callbacks/5second/system/timeseries.lua][...ff/graft/repos/ntopng/scripts/lua/modules/ntop_utils.lua:342: global 'ends' is not callable (a nil value)]
^C04/Sep/2022 21:53:46 [main.cpp:50] Shutting down...
04/Sep/2022 21:53:46 [NetworkInterface.cpp:3080] Flow alerts dump thread terminated for enp2s0
04/Sep/2022 21:53:47 [NetworkInterface.cpp:3138] Host alerts dump thread terminated for enp2s0
04/Sep/2022 21:53:47 [PeriodicActivities.cpp:100] Terminated periodic activites...
04/Sep/2022 21:53:47 [PcapInterface.cpp:334] Terminated packet polling for enp2s0
04/Sep/2022 21:53:50 [main.cpp:438] Terminating...
04/Sep/2022 21:53:50 [ProtoStats.cpp:35] [IPv4]  10.90 KB/83.00 Packets
04/Sep/2022 21:53:50 [ProtoStats.cpp:35] [IPv6]  0 B/0.00 Packets
04/Sep/2022 21:53:50 [ProtoStats.cpp:35] [ARP]   306 B/6.00 Packets
04/Sep/2022 21:53:50 [ProtoStats.cpp:35] [MPLS]  0 B/0.00 Packets
04/Sep/2022 21:53:50 [ProtoStats.cpp:35] [Other] 540 B/9.00 Packets
04/Sep/2022 21:53:50 [Ntop.cpp:2888] Polling shut down [interface: enp2s0]
04/Sep/2022 21:53:50 [Ntop.cpp:2908] Executing shutdown script [shutdown.lua]
04/Sep/2022 21:53:51 [HTTPserver.cpp:1709] HTTP server terminated
04/Sep/2022 21:53:51 [NetworkInterface.cpp:3298] Cleanup interface enp2s0
04/Sep/2022 21:53:51 [AddressResolution.cpp:54] Address resolution stats [2 resolved][0 failures]
```

An example unit test failure  (because of warnings in NTOPS log)

```
[jeff@rocky01 rest]$ ./run.sh --test=v2/alert_tls_certificate_selfsigned
[i] Connectivity ok
[>] Running test 'v2/alert_tls_certificate_selfsigned' (1/1)
[!] ntopng Error: ntopng generated errors or warnings running 'v2/alert_tls_certificate_selfsigned'
04/Sep/2022 21:40:05 [LuaEngine.cpp:595] WARNING: Script failure [/home/jeff/graft/repos/ntopng/scripts/callbacks/5second/system/timeseries.lua][...ff/graft/repos/ntopng/scripts/lua/modules/ntop_utils.lua:342: global 'ends' is not callable (a nil value)]
04/Sep/2022 21:40:10 [LuaEngine.cpp:595] WARNING: Script failure [/home/jeff/graft/repos/ntopng/scripts/callbacks/5second/system/timeseries.lua][...ff/graft/repos/ntopng/scripts/lua/modules/ntop_utils.lua:342: global 'ends' is not callable (a nil value)]
04/Sep/2022 21:40:15 [LuaEngine.cpp:595] WARNING: Script failure [/home/jeff/graft/repos/ntopng/scripts/callbacks/5second/system/timeseries.lua][...ff/graft/repos/ntopng/scripts/lua/modules/ntop_utils.lua:342: global 'ends' is not callable (a nil value)]
04/Sep/2022 21:40:20 [LuaEngine.cpp:1254] WARNING: Script failure [/home/jeff/graft/repos/ntopng/scripts/lua/rest/v2/get/flow/alert/list.lua][...graft/repos/ntopng/scripts/lua/modules/lua_utils_gui.lua:308: global 'hostdetails_exists' is not callable (a nil value)]
04/Sep/2022 21:40:20 [LuaEngine.cpp:595] WARNING: Script failure [/home/jeff/graft/repos/ntopng/scripts/callbacks/5second/system/timeseries.lua][...ff/graft/repos/ntopng/scripts/lua/modules/ntop_utils.lua:342: global 'ends' is not callable (a nil value)]
[!] ntopng TESTS completed with errors: 0 out of 1 completed successfully.
```

**Comment by cardigliano** [comment]
The same test seems to be working in our environment, we need to dig a bit more

[>] Running test 'v2/alert_tls_certificate_selfsigned' (1/1)
[i] OK
[>] ntopng TESTS completed successfully

**Comment by MatteoBiscosi** [comment]
Hi @dandare100 the issue should be fixed thanks to luca's commit dbf6e0b531aa9f63a5adbc9015e96d7b2c4d99ad

**Comment by dandare100** [comment]
@MatteoBiscosi @cardigliano 
This is now sorted and fixed.
Thank you very much
